src/components/Container.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styled from "styled-components";
+import profiles from '../data/profiles';
+import ProfileList from './ProfileList';
+
+export default function Container() {
+    return (
+        <Wrapper>
+            <Title>CEOS 프론트엔드 운영진</Title>
+            <ProfileList />
+        </Wrapper>
+    );
+}
+
+const Wrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+    background-color: rgb(230, 255, 255);
+    padding: 5rem 10rem;
+`;
+
+const Title = styled.div`
```

Comment:
태그 안에 글을 써야 할 때에는 div 태그보다 p 태그가 더 적절합니다! :)
```suggestion
const Title = styled.p`
```